crates/polars-ops/src/series/ops/cum_agg.rs

```diff
@@ -8,16 +8,16 @@ use polars_core::prelude::*;
 use polars_core::series::IsSorted;
 use polars_core::utils::{CustomIterTools, NoNull};
 use polars_core::with_match_physical_numeric_polars_type;
+use polars_utils::float::IsFloat;
+use polars_utils::min_max::MinMax;
 
 fn det_max<T>(state: &mut T, v: Option<T>) -> Option<Option<T>>
 where
-    T: Copy + PartialOrd + AddAssign + Add<Output = T>,
+    T: Copy + PartialOrd + AddAssign + Add<Output = T> + MinMax,
```

Comment:
One final nitpick, can you minimize these trait bounds? As far as I can see this now only requires `Copy` and `MinMax`. The rest looks good :)